src/app/account/login/login.component.html

```diff
@@ -1,3 +1 @@
-<p>
-  login works!
-</p>
+<button [routerLink]="['/account/user', 'id']">GO FFFf</button>
```

Comment:
@PsihovMaster make buttons with Logic e.g: Navigate to profile; 
"GO FFFf" was only for example don't never forget to name all with logic